docs/octopus-rest-api/octopus.client.md

```diff
@@ -4,54 +4,71 @@ description: Octopus.Client is an open source .NET library that makes it easy to
 position: 110
 ---
 
-Octopus.Client is an [open source](https://github.com/OctopusDeploy/OctopusClients) .NET library that makes it easy to write C# programs that manipulate the [Octopus Deploy REST API](/docs/octopus-rest-api/index.md).
+Octopus.Client is an [open source](https://github.com/OctopusDeploy/OctopusClients) .NET library that makes it easy to write C# programs or PowerShell scripts that manipulate the [Octopus Deploy REST API](/docs/octopus-rest-api/index.md).
 
-Because the Octopus Deploy application itself is built entirely on the API, C# programs using the API can do anything that could be done by a user of the application itself.
+Because the Octopus Deploy application itself is built entirely on the API, any programming language that can make HTTP requests to the API can do anything that could be done by a user of the application itself.
 
-The NuGet package contains both a .NET Framework build as well as a .NET Standard build. The .NET Framework build targets 4.5 or later and contains both the synchronous and asynchronous API. The .NET Standard build is compatible with a [variety of runtimes](https://docs.microsoft.com/en-us/dotnet/articles/standard/library), including .NET Core 1.0 and only contains the asynchronous API.
+Octopus.Client is [published on NuGet](https://www.nuget.org/packages/Octopus.Client). The package contains both a .NET Framework build as well as a .NET Standard build. The .NET Framework build targets 4.5 or later, while the .NET Standard build is cross-platform and compatible with a [variety of runtimes](https://docs.microsoft.com/en-us/dotnet/articles/standard/library), including .NET Core and can be used from PowerShell Core.
 
 ## Getting Started {#Octopus.Client-Gettingstarted}
 
 :::hint
-The complete details for the API itself - where to find it, how to authenticate, the available resources and so-on - are available at [the API documentation site](http://g.octopushq.com/ApiDocs).
+Details for the API itself - where to find it, how to authenticate, the available resources and so-on - are available at [the API documentation site](https://g.octopushq.com/ApiDocs), while [swagger documentation](https://demo.octopus.com/swaggerui) is also available
 :::
 
-To use the C# client, first install the package via NuGet:
+To use from PowerShell, use the `Install-Package` command from the Microsoft [PackageManagement](https://docs.microsoft.com/en-us/powershell/module/packagemanagement) module:
+```powershell
+Install-Package Octopus.Client -source https://www.nuget.org/api/v2
+$path = Join-Path (Get-Item ((Get-Package Octopus.Client).source)).Directory.FullName "lib/net45/Octopus.Client.dll"
+Add-Type -Path $path
+```
+_This is a bit harder than it should be, but [there are plans](https://github.com/PowerShell/PowerShell/issues/6724) to make it simpler._
```

Comment:
I prefer not to include promises about future states in the docs. We should try to stick to how to solve the problem now, and when there's a better way, add that to the docs. (People often put these statements in and then forget to update them, or future plans get dropped).